src/app/_components/layout/sidebar.css

```
@@ -16,7 +16,19 @@
 }
 
 /* ============================================================
-   Create Action — soft tinted CTA
+   Main content offset - responds to data-sidebar on <html>
+   ============================================================ */
+@media (min-width: 640px) {
```

Stylelint (17.13.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)